clients/web/src/lib/components/actions/Tooltip/tooltip.postcss

```diff
@@ -0,0 +1,25 @@
+.tooltip {
```

Comment:
Are these styles global? If so, we may need a more unique class name than `tooltip`. I could see that being duplicated somewhere else in the app

Reply:
Not sure what other context `tooltip` would be showing up in, but I believe this is global